src/components/application_manager/test/commands/mobile/system_request_test.cc

```diff
@@ -1,385 +0,0 @@
-      mobile_apis::RequestType::QUERY_APPS;
-  (*msg_)[am::strings::msg_params][am::strings::file_name] = file_name_;
-  (*msg_)[am::strings::msg_params][am::strings::request_type] = request_type;
-
-  // json_data was not contain required parameter, `json::response`
-  const std::string json_data("{\"invalid_json_data\":{}}");
-  const std::vector<uint8_t> binary_data(json_data.begin(), json_data.end());
-  (*msg_)[am::strings::params][am::strings::binary_data] = binary_data;
-  CommandPtr command(CreateCommand<SystemRequest>(msg_));
-
-  ON_CALL(policy_handler_mock, IsRequestTypeAllowed(kPolicyAppId, request_type))
-      .WillByDefault(Return(true));
-  ON_CALL(mock_app_manager_, application(kConnectionKey))
-      .WillByDefault(Return(app_));
-  ON_CALL(mock_app_manager_settings_, system_files_path())
-      .WillByDefault(ReturnRef(kBinaryFolder));
-  ON_CALL(mock_app_manager_settings_, app_storage_folder())
-      .WillByDefault(ReturnRef(kStorageFolder));
-
-  ON_CALL(mock_app_manager_, SaveBinary(_, kBinaryFolder, file_name_, 0))
-      .WillByDefault(Return(mobile_result::SUCCESS));
-
-  EXPECT_CALL(
-      mock_app_manager_,
-      ManageMobileCommand(MobileResultCodeIs(mobile_result::GENERIC_ERROR), _));
-
-  command->Run();
-}
-
-TEST_F(SystemRequestTest, Run_ValidQueryAppData_SUCCESS) {
-  const mobile_apis::RequestType::eType request_type =
-      mobile_apis::RequestType::QUERY_APPS;
-  (*msg_)[am::strings::msg_params][am::strings::file_name] = file_name_;
-  (*msg_)[am::strings::msg_params][am::strings::request_type] = request_type;
-
-  std::ostringstream json_data_stream;
-  json_data_stream << "{\"" << am::json::response << "\":{}}";
-
-  // json_data should contain `json::response` parameter, to be valid
-  const std::string json_data(json_data_stream.str());
-  const std::vector<uint8_t> binary_data(json_data.begin(), json_data.end());
-
-  (*msg_)[am::strings::params][am::strings::binary_data] = binary_data;
-  CommandPtr command(CreateCommand<SystemRequest>(msg_));
-
-  ON_CALL(policy_handler_mock, IsRequestTypeAllowed(kPolicyAppId, request_type))
-      .WillByDefault(Return(true));
-  ON_CALL(mock_app_manager_, application(kConnectionKey))
-      .WillByDefault(Return(app_));
-  ON_CALL(mock_app_manager_settings_, system_files_path())
-      .WillByDefault(ReturnRef(kBinaryFolder));
-  ON_CALL(mock_app_manager_settings_, app_storage_folder())
-      .WillByDefault(ReturnRef(kStorageFolder));
-
-  ON_CALL(mock_app_manager_, SaveBinary(_, kBinaryFolder, file_name_, 0))
-      .WillByDefault(Return(mobile_result::SUCCESS));
-
-  EXPECT_CALL(mock_app_manager_, ProcessQueryApp(_, kConnectionKey));
-
-  EXPECT_CALL(
-      mock_app_manager_,
-      ManageMobileCommand(MobileResultCodeIs(mobile_result::SUCCESS), _));
-
-  command->Run();
-}
-
-}  // namespace system_request
-}  // namespace mobile_commands_test
-}  // namespace commands_test
-}  // namespace components
-}  // namespace test
```

Comment:
@LevchenkoS it was pushed in this commit by mistake. UTs for this command will be included in PR for the related issue.